languages/en.ini

```diff
@@ -606,6 +606,8 @@ ils_offline_login_message = "Your account details will be unavailable during thi
 ils_offline_status = "Our Library Management System is currently under maintenance."
 ils_offline_title = "System Under Maintenance"
 ils_transaction_history_disabled = "Loan history is not enabled for the active library card."
+Image = "Image"
+Image or Video = "Image or Video"
```

Comment:
I don't think I like the addition of an "Image or Video" facet, because that doesn't really make much sense from a faceting perspective. Maybe it's better to just map it to Video, with a comment that it may not be appropriate, and let people address the ambiguities locally. Open to other suggestions -- I just think it's not going to help the user much to have a facet value that could be one of two other categories, but doesn't group with either of them.

Reply:
Thanks @demiankatz . That's fine and agree this is clunky. I'll map ProjectedMedium to "Slide" (i think this a better default than Video and seems to have been used at some point in the past) in translation_map and leave it out of en.ini/ga.ini? The only alternative would be to have a user-facing string of "Projected Medium" but I don't think that'd make much sense to most users.

Reply:
Yes, sounds good to me -- mapping to Slide matches past precedent, as you say.

Reply:
done in latest commit